conda_env/specs/requirements.py

```diff
@@ -0,0 +1,43 @@
+import os
+from .. import env
+
+
+class RequirementsSpec(object):
+    '''
+    Reads depedencies from a requirements.txt file
+    and returns an Environment object from it.
+    '''
+    msg = None
+
+    def __init__(self, filename=None, name=None, **kwargs):
+        self.filename = filename
+        self.name = name
+        self.msg = None
+
+    def can_handle(self):
```

Comment:
It would be nice to have normal methods after private ones...

so

```python
def __init__  ...

def _valid_file ...

def _valid_name ....

def can_handle ...
```